REQUIRE

```diff
@@ -1,8 +1,8 @@
-julia 0.4
+julia 0.5
 Cairo
 Colors
-IniFile
-Tk
-Compat 0.7.16
+IniFile 0.3.1
+Tk 0.5.1
```

Comment:
Unclear, since Tk wasn't being tested on macOS at all, so it may well have been failing there on 0.5 as well.